exercises/webapp/src/components/HackerNews.spec.js

```diff
@@ -0,0 +1,70 @@
+import {
+  mount
+} from '@vue/test-utils';
+import HackerNews from './HackerNews.vue';
+import NewsItem from './NewsItem.vue';
+
+describe("Sorting Order", () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(HackerNews, {
+      propsData: {
+        initialNews: [{
+          id: 0,
+          title: "Message 1",
+          voteCount: 2
+        }, {
+          id: 1,
+          title: "Message 2",
+          voteCount: 4
+        }],
+      }
+    });
+  });
+
+  it("sorting order", async () => {
```

Comment:
```suggestion
describe('"Reverse Order" button', () => {
  it("toggles sort orrder", async () => {/* ... */}
})
```
Try to write test case descriptions so that your co-workers can understand the behaviour of the component.